stactools_threedep/stactools/threedep/stac.py

```diff
@@ -0,0 +1,48 @@
+from typing import Optional
+
+from pystac import Item
+from shapely.geometry import shape
+
+from stactools.core.io import ReadHrefModifier
+from stactools.threedep.metadata import Metadata
+from stactools.threedep.constants import DEFAULT_BASE
+
+
+def create_item(href: str,
+                read_href_modifier: Optional[ReadHrefModifier] = None,
+                base: str = DEFAULT_BASE) -> Item:
+    """Creates a STAC item from an href to an XML metadata file."""
+    metadata = Metadata.from_href(href, read_href_modifier)
+    return create_item_from_metadata(metadata, base)
+
+
+def create_item_from_product_and_id(product: str,
+                                    id: str,
+                                    base: str = DEFAULT_BASE) -> Item:
+    """Creates a STAC item from a product (e.g. "1") and an ID (e.g. "n41w106")."""
+    metadata = Metadata.from_product_and_id(product, id)
+    return create_item_from_metadata(metadata, base)
+
+
+def create_item_from_metadata(metadata: Metadata, base: DEFAULT_BASE) -> Item:
+    """Creates a STAC item from Metadata."""
+    geometry = metadata.geometry
+    bbox = list(shape(geometry).bounds)
+    item = Item(id=metadata.stac_id,
+                geometry=geometry,
+                bbox=bbox,
+                datetime=metadata.datetime,
+                properties={})
+    start_datetime = metadata.start_datetime
+    end_datetime = metadata.end_datetime
+    if start_datetime and end_datetime:
+        item.common_metadata.start_datetime = start_datetime
+        item.common_metadata.end_datetime = end_datetime
+    item.common_metadata.gsd = metadata.gsd
```

Comment:
I'm sure you considered the other common metadata parameters, but I was wondering if there was any info about the specific platform or instrument used to collect any given piece of data.
Seems like `mission` could be 3dep, but it's a bit redundant since the collection already encapsulated the whole mission.

https://github.com/radiantearth/stac-spec/blob/master/item-spec/common-metadata.md#instrument

Reply:
I'll give it another passthrough, but in general it doesn't seem to make sense to provide more specific platform information since the data is relatively mixed-provenance. If we end up supporting focus-region data (e.g. 1m DEMs and such) then platform and friends should make more sense.